src/masternodes/rpc_vault.cpp

```diff
@@ -490,7 +491,7 @@ UniValue listvaults(const JSONRPCRequest& request) {
             return false;
         }
 
-        if (isMine && !(IsMineCached(*pwallet, data.ownerAddress) & ISMINE_SPENDABLE)) {
+        if (isMine && !(IsMineCached(*pwallet, data.ownerAddress) & ISMINE_ALL)) {
```

Comment:
It should be `ISMINE_SPENDABLE` we use it all the way, it's better to not differentiate here or to include watch only elsewhere.